src/htmlgen.cpp

```diff
@@ -420,6 +420,8 @@ static QCString substituteHtmlKeywords(const QCString &str,
     if (!serverBasedSearch)
     {
       searchCssJs += "<script type=\"text/javascript\" src=\"$relpath^search/searchdata.js\"></script>\n";
+    } else {
+      searchCssJs += "<link rel=\"stylesheet\" href=\"https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0\" />";
```

Comment:
I don't really like the external dependency. May break in the future and prevents off-line browsing. Better to bundle the set of icons with doxygen.

Reply:
Also privacy concerns and legal headaches that come with it, depending on the country of residence. Please don't load external resources.